src/Sticky.ts

```diff
@@ -14,7 +14,7 @@ export class Sticky {
         // if (!useStickyBookmarks) {
         //     return false;
         // }
-
+        let keepOnDelete = vscode.workspace.getConfiguration("numberedBookmarks").get("keepBookmarksOnDelete", false);
```

Comment:
Add type definition to `keepOnDelete` variable (`let keepOnDelete: boolean = ...`